<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 202, 32);</string>
  </property>
  <widget class="QPushButton" name="otdel">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>588</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt;color:rgb(243, 243, 243)</string>
   </property>
   <property name="text">
    <string>Отделы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="what">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>224</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>142</y>
     <width>108</width>
     <height>144</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="open">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
      </property>
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="apdate">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="what_5">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
      </property>
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dell">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="typdoc_2">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>143</y>
     <width>210</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font-size:15pt; color:rgb(243, 243, 243)</string>
   </property>
   <item>
    <property name="text">
     <string>Поиск</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="whattext">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>182</y>
     <width>210</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; 
color:rgb(0, 0, 0);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>6</y>
     <width>172</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font-size:28pt; color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Связанные&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="docs">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>588</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
   <property name="text">
    <string>Документы</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twStaffs">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>294</y>
     <width>726</width>
     <height>273</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; 
color:rgb(243, 243, 243);
background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="client">
   <property name="geometry">
    <rect>
     <x>468</x>
     <y>588</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
   <property name="text">
    <string>Сотрудники</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>11</y>
     <width>208</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font-size:28pt; color:rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Исполнители&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="docs3">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>66</y>
     <width>322</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; 
background-color: rgb(0, 0, 0);
color:rgb(243, 243, 243)</string>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>Документы</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="sotrudnic">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>104</y>
     <width>322</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; 
background-color: rgb(0, 0, 0);
color:rgb(243, 243, 243)</string>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>Сотрудники</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>143</y>
     <width>108</width>
     <height>144</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="open_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
      </property>
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="apdate_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="what_7">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
      </property>
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dell_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="whattext_2">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>182</y>
     <width>210</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; 
color:rgb(0, 0, 0);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="typdoc_3">
   <property name="geometry">
    <rect>
     <x>545</x>
     <y>143</y>
     <width>210</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font-size:15pt; color:rgb(243, 243, 243)</string>
   </property>
   <item>
    <property name="text">
     <string>Поиск</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="what_2">
   <property name="geometry">
    <rect>
     <x>623</x>
     <y>224</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QComboBox" name="docs1">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>64</y>
     <width>322</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; 
background-color: rgb(0, 0, 0);
color:rgb(243, 243, 243)</string>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>Главный документ</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="docs2">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>102</y>
     <width>322</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; 
background-color: rgb(0, 0, 0);
color:rgb(243, 243, 243)</string>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>Связанный документ</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
